docs/resources/job.md

```diff
@@ -209,6 +210,18 @@ One of the `query`, `dashboard` or `alert` needs to be provided.
 * `on_start` - (Optional) (List) list of emails to notify on failure
 * `on_success` - (Optional) (List) list of emails to notify on failure
 
+### webhook_notifications Configuration Block
+
+Each entry in `webhook_notification` block takes a list `webhook` blocks. The field is documented below.
+
+* `on_failure` - (Optional) (List) An optional list of system notification IDs to call when the run fails. A maximum of 3 destinations can be specified for the `on_failure` property.
+* `on_start` - (Optional) (List) An optional list of system notification IDs to call when the run starts. A maximum of 3 destinations can be specified for the `on_start` property.
+* `on_success` - (Optional) (List) An optional list of system notification IDs to call when the run completes successfully. A maximum of 3 destinations can be specified for the `on_success` property.
+
+### webhook Confiuration Block
+
+* `id` - ID of the destination that is notified when an event defined in `webhook_notifications` is triggered.
```

Comment:
there is a discrepancy in terminology - here the `destination` word is used, and in other places: `system notification ID` - it's better to standartize

Reply:
Changed, ty!